packages/validate-icu-locales/package.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "@scaleway/validate-icu-locales",
+  "version": "1.0.0",
+  "description": "A small cli to handle icu errors on locales files",
+  "keywords": [
+    "icu",
+    "i18n",
+    "cli"
+  ],
+  "type": "module",
+  "bin": "dist/index.js",
```

Maybe 

```suggestion
  "bin": {
    "validate-icu": "dist/index.js"
   },
```

A tad weird to have scoped packages as cli

---

agree